pkgs/ffigen/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
   values are handled properly, and are equal to each other in Dart as well.
 - Rename ObjC interface methods that clash with type names. Fixes
   https://github.com/dart-lang/native/issues/1007.
+- __Breaking change__: Update Enum types to use the integer type returned by
+  libclang instead of `Int32`. The int type used is unsigned, but converted
+  to signed if any enum value is negative. Added config `silence-enum-warning`
```

Comment:
This doesn't mention why it's a warning. Maybe add "Enum integer types are implementation-defined and not part of the ABI. Therefore FFIgen does a best-effort approach trying to mimic the most common compilers for the various OS and architecture combinations. To silence the warning: ..."

Reply:
done